app/assets/stylesheets/application.scss

```diff
@@ -17,5 +17,48 @@
 @import "bootstrap";
 
 body {
-  background-color: white;
+  background-color: #e5e8e0;
```

Comment:
Color literals like `#e5e8e0` should only be used in variable declarations; they should be referred to via variable everywhere else.